templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -132,7 +132,14 @@ class IKFastKinematicsPlugin : public kinematics::KinematicsBase
   /** @class
    *  @brief Interface for an IKFast kinematics plugin
    */
-  IKFastKinematicsPlugin():active_(false){}
+  IKFastKinematicsPlugin():
+    active_(false)
+  {
+    srand( time(NULL) );
```

Comment:
We have applications where we load ikfast through the plugin directly.  In such cases the parent process may or may not initialize the random seed.  As far as I know there's no penalty for initializing it more than once.